sandboxes/dynamicanalysis/analyze-python.py

```diff
@@ -85,33 +86,46 @@ def import_single_module(import_path):
 
 def import_module(import_path):
     print('Importing', import_path)
+    # noinspection PyBroadException
     try:
-        module = importlib.import_module(import_path)
-    except:
+        importlib.import_module(import_path)
+    except Exception:
```

Comment:
This should probably still be a bare `except:` to prevent the script from aborting unexpectedly.

Reply:
Okay. Bare `except` is equivalent to `except BaseException` so I'll put that to make it more explicit, and add a comment